packages/fx-core/src/plugins/solution/fx-solution/v2/executeUserTask.ts

```diff
@@ -1028,7 +1033,17 @@ export async function addSso(
     );
   }
 
-  return ok(undefined);
+  ctx.telemetryReporter.sendTelemetryEvent(SolutionTelemetryEvent.AddSso, {
```

Comment:
There's only AddSso-start and AddSso(success), AddSso(fail) is also needed for error case.